### ISSUE TITLE
Cannot find type `TextFormElement` in scope - Error

### ISSUE THREAD

**Issue by kwylez** [Esri/arcgis-maps-sdk-swift-toolkit#905]
I'm currently incorporating the toolkit in my own SPM as a dependency and started to test out the latest beta version of Xcode 16.1 (beta 3). However I'm getting the following error when trying to compile:

Cannot find type `TextFormElement` in the `FeatureFormView` I'm using the latest `v.latest` branch.

<img width="383" alt="image002" src="https://github.com/user-attachments/assets/d5d20307-a04a-4fca-9d80-6f44e3b146f9">
<img width="1309" alt="image001" src="https://github.com/user-attachments/assets/ded3a871-6a9e-4d46-877c-9879616bdcab">

**Comment by mhdostal** [comment]
@kwylez Thank you for your interest in the Swift Toolkit! I'd be interested to hear about what you're building with it.

The reason for your issue is that the `v.next` branch of the Toolkit is for functionality coming in the next release of the Toolkit and it depends upon unreleased functionality in the [ArcGIS Maps SDK for Swift](https://github.com/Esri/arcgis-maps-sdk-swift/). The release is currently planned for before the end of the year, for both the SDK and Toolkit (they are released in tandem).

Was there specific functionality in `v.next` that you were looking to use?

**Comment by kwylez** [comment]
@mhdostal Thank you for your quick reply! I'm building some features for an internal app for Duke Energy. I can't disclose very much but the arcgis framework(s) have been a pleasure to work with.

I wasn't looking for any specific functionality in the `v.next` branch, rather any updates that would fix compile errors that I got with moving my framework over to Swift 6 and using Xcode 16.1 beta 2. After upgrading to beta 3 I got the above compile error which was "new". However, when I switched the ArcGIS dependency _back_ to using the latest release tag and  beta all my compile errors were resolved. 

In summary, it looks like it was an issue with Apple / Xcode Beta 2...nothing with the framework.

**Comment by mhdostal** [comment]
@kwylez - Excellent, I'm glad your issue is resolved! Let us know if there's anything else you need!
